website/data/docs-nav-data.json

```diff
@@ -1821,6 +1821,10 @@
         "title": "Admin Partitions",
         "path": "enterprise/admin-partitions"
       },
+      {
```

Comment:
Could we move this to the bottom of the Enterprise section? It should be the last entry there.